libs/SalesforceReact/src/com/salesforce/androidsdk/reactnative/app/SalesforceReactSDKManager.java

```diff
@@ -72,9 +72,10 @@ private static void init(Context context, Class<? extends Activity> mainActivity
     		INSTANCE = new SalesforceReactSDKManager(context, mainActivity, loginActivity);
     	}
 
+		initInternal(context);
```

Comment:
This call is meant to be after the upgrade step, by design. There's a lot of internal initialization that occurs there, and without running upgrade steps, it would lead to permanent corruption. For example, `SmartStore` DB opening calls. Let's move this back below.